app/views/About.js

```diff
@@ -23,18 +25,60 @@ class AboutScreen extends Component {
   }
 
   handleBackPress = () => {
+    this.setState({ tapCount: 0 });
     this.backToMain();
     return true;
   };
 
   componentDidMount() {
+    GetStoreData(DEBUG_MODE).then(dbgMode => {
+      if (dbgMode == 'true') {
+        this.setState({ tapCount: 4 });
+      }
+    });
+
     BackHandler.addEventListener('hardwareBackPress', this.handleBackPress);
   }
 
   componentWillUnmount() {
     BackHandler.removeEventListener('hardwareBackPress', this.handleBackPress);
   }
 
+  handleTapTeam = () => {
+    this.setState({ tapCount: this.state.tapCount + 1 });
+    if (this.state.tapCount >= 3) {
+      if (this.state.tapCount == 3) {
+        // Debug mode on
+        SetStoreData(DEBUG_MODE, 'true');
+
+        // Create faux intersection data
+        let pseudoBin = [];
+        for (let i = 0; i < 28; i++) {
+          const intersections = Math.max(
+            0,
+            Math.floor(Math.random() * 50 - 20),
+          );
+          pseudoBin.push(intersections);
+        }
+        let dayBin = JSON.stringify(pseudoBin);
+        SetStoreData(CROSSED_PATHS, dayBin);
+      } else if (this.state.tapCount == 7) {
```

Comment:
Suggestion... instead of counting more taps to disable:

Add a label next to the Team icon or title:

`Demo mode is enabed, tap to disable`

Now its very clear if you're in or out of demo mode.

Reply:
I added a label.  It is English only, but I think that is reasonable for a hidden debug/demo feature.